### DISCUSSION TITLE
Softpot Membrane Potentiomoter

### DISCUSSION THREAD

**Discussion by egolikeness** [shanteacontrols/OpenDeck#85] [Q&A]
Hey,
Love the product!
I am very new at this whole thing, so forgive me if I missed something in the documentation.
I would like to hook up one of these https://www.sparkfun.com/products/8681
I can get it to work, but it seems to only output the signals 6 or 127 depending on where I touch it.
Am I doing something wrong?

Also, re RGB leds. The wiki says it has to be enabled in software.
The code is there, but I have no idea how to do that.
Is there a tutorial somewhere that explains it exactly how to do it from start to finish?

Thanks!

**Comment by paradajz** [comment]
Thanks.

The potentiometer you've linked is FSR. You need to select FSR as the type in analog section in the configurator. Check the wiki on how to properly connect it to the board. Note that currently FSR has limited support in firmware and can only react on initial press. I'll probably improve this in next releases.

As for RGB LEDs, follow the hookup guide and just toggle RGB LED option in the configurator, that is all.

**Reply by egolikeness** [reply]
Thanks! Is there a different ribbon potentiometer you would recommend?

**Reply by paradajz** [reply]
That one is just fine.

**Reply by paradajz** [reply]
Sometimes I speak/write before I should. I just noticed that the pot you've linked actually has three pins, so it should behave/be connected just like normal pot actually. Unfortunately I don't have one around here to test how it behaves.

**Reply by egolikeness** [reply]
Ok, thank you!
So, for my next question, I'm trying to set up a Nexton NX4932K035 and I can't seem to get my data on it.
I tried it using an SD card, Fat32 and when I put it in I get SD Card INIT Error please check your sd card is fat32.
And when I try to do it through the open deck directly I get "Connection Failed forced interrupt."
Any thoughts?

I've got something really interesting going on here, and I think you will get a kick out of it once it's done. 
Thanks!

**Reply by paradajz** [reply]
If you're using SD card, the card needs to have .TFT file in root directory, exported from their GUI. If you're sing OpenDeck to flash the display, you need to reboot the board into CDC mode from online configurator.

**Reply by egolikeness** [reply]
I saw that, based on my experience and what I could glean from the web, it only works with 16gig cards. Not 32. 
Just FYI.

**Reply by egolikeness** [reply]
I'm trying to hook these buttons up, but I'm not entirely sure if it can be done and how.
Any thoughts?
https://www.sparkfun.com/products/9277

**Comment by kimgalibert** [comment]
i have two of the smaller softpot membrane potentiometers which i'd like to use for pitchbend or midi cc.  both are transmitting data nonstop when configured as potentiometers.  i'll test again shortly with them configured as fsr's, although i'm not sure that will provide the desired control.

should the membrane pots be attached in the same way a regular ones?  do i need to add a capacitor or something?

thanks

**Reply by paradajz** [reply]
It's hard for me to say what's going on since I don't have any around here. I'll be ordering some components soon so I'll get few of those pots and probably improve handling of them.

**Reply by kimgalibert** [reply]
i've done some further testing and i don't believe the softpots will be an effective option.  of the two i installed, one was flaky and one got hot enough to melt the plastic.  unlike the fsr's they don't transmit continuously when pressed, and so if used for pitch control they've very jumpy.  

i found a blog which discusses the heat issue, it appears that if you press down in two places or in the wrong place the unit can overheat.

i would like to thank you for this software, it's a treasure.  having easily programmable buttons and knobs and sliders wherever i need them is amazing.
